plugins/defaultplugins/l3plugin/model/l3/l3.proto

```diff
@@ -3,17 +3,17 @@ syntax = "proto3";
 package l3;
 
 message StaticRoutes {
-    message Ip {
+    message Route {
         uint32 vrf_id = 1;                   /* VRF identifier */
```

Comment:
Lukas you wanted to write 'it can't be removed', right? If it is necessary for rc?